shared/types.ts

```diff
@@ -33,3 +33,38 @@ export interface NodeError {
   msg: string;
   stackTrace: string;
 }
+
+interface NetSettings {
+  netId: number;
+  netName: string;
+  explorerUrl: string;
+  dashUrl: string;
+  minCommitmentSize: number;
+  layerDurationSec: number;
+  genesisTime: string;
+}
+
+interface Account {
+  [key: string]: {
+    account: {
```

Comment:
`Account` should be the type of one element.
BTW, take a look at `/app/types/wallet.ts` is already contains an `Account` type.
Probably it should be moved to the `shared` directory.